superset-frontend/src/components/ListView/CardCollection.tsx

```diff
@@ -0,0 +1,61 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import { TableInstance } from 'react-table';
+import styled from '@superset-ui/style';
+
+interface Props {
+  renderCard?: (row: any) => React.ReactNode;
+  prepareRow: TableInstance['prepareRow'];
+  rows: TableInstance['rows'];
+  loading: boolean;
+}
+
+const CardContainer = styled.div`
+  justify-content: space-between;
```

Comment:
This looks fine when there are 3+ columns, but with 2 columns the cards look like they're pushed to the edges of the container:
<img width="1380" alt="Screen Shot 2020-08-07 at 11 42 57 AM" src="https://user-images.githubusercontent.com/8216382/89677996-3fceae80-d8a3-11ea-8693-e4bb1d2c7b60.png">

Using `justify-content: space-around` would space them more evenly:
<img width="1383" alt="Screen Shot 2020-08-07 at 11 44 40 AM" src="https://user-images.githubusercontent.com/8216382/89678088-6e4c8980-d8a3-11ea-9afe-fdcf59b13fe5.png">

Reply:
I ended up ditching the flexbox implementation in favor of CSS grid. I really didn't like how this was sometimes happening on the last row

<img width="1844" alt="Screen Shot 2020-08-07 at 2 24 20 PM" src="https://user-images.githubusercontent.com/10255196/89689683-c4c4c280-d8b9-11ea-861f-ee3b6aaaaeb0.png">